controllers/csiaddons/persistentvolumeclaim_controller.go

```diff
@@ -64,6 +64,7 @@ const (
 	defaultSchedule = "@weekly"
 )
 
+//+kubebuilder:rbac:groups=coordination.k8s.io,resources=leases,verbs=get
```

Comment:
```suggestion
```
We don't need to find leader in pvc controller right ?

Reply:
we do, because offline reclaim-space needs to run on the leading controller